pagerduty/data_source_pagerduty_service.go

```diff
@@ -19,6 +19,49 @@ func dataSourcePagerDutyService() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"auto_resolve_timeout": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"acknowledgement_timeout": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"alert_creation": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"escalation_policy": {
+				Type:     schema.TypeMap,
+				Computed: true,
+				// Using the `Elem` block to define specific keys for the map is currently not possible.
+				// The workaround described in SDK documentation is to confirm the required keys are set when expanding the Map object inside the resource code.
+				// See https://www.terraform.io/docs/extend/schemas/schema-types.html#typemap
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"teams": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "List of users who are members of the team",
```

Comment:
This description doesn't look right -- matching the API docs it should be `The set of teams associated with the service`

Reply:
Fixed

Reply:
Thanks